backend/app/services/project_automations.py

```
@@ -350,6 +350,73 @@ async def run_now(
+    async def run_for_workflow_node(
+        self,
+        db: Session,
+        project_id: str,
+        automation_id: str,
+        item_id: str,
+        workflow_node_id: str,
+        user_id: int,
+    ) -> dict:
+        require_cloud_project_role(db, project_id, user_id, BaseRole.Developer)
+        rule = self._rule(db, project_id, automation_id)
+        item = loop_item_service.get(db, item_id, user_id)
```

Ruff (0.16.1)

[error] Undefined name `loop_item_service`

(F821)